From 5f3577dce4b3feeadecf1cbe98cd073713e1136e Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Sat, 23 May 2020 09:03:38 +0200
Subject: [PATCH 2/2] Nano emmc defconfig: add necessary configs

photon-nano uses the emmc u-boot defconfig
so it needs to add the hup related  configs.

Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 configs/p3450-0002_defconfig | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/configs/p3450-0002_defconfig b/configs/p3450-0002_defconfig
index 18b57f5f4f..a9480e0536 100644
--- a/configs/p3450-0002_defconfig
+++ b/configs/p3450-0002_defconfig
@@ -21,7 +21,7 @@ CONFIG_CMD_SF=y
 CONFIG_CMD_SPI=y
 CONFIG_CMD_USB=y
 CONFIG_CMD_USB_MASS_STORAGE=y
-# CONFIG_CMD_SETEXPR is not set
+CONFIG_CMD_SETEXPR=y
 # CONFIG_CMD_NFS is not set
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_OF_LIVE=y
@@ -57,3 +57,8 @@ CONFIG_ENV_OFFSET=0xFFFFE000
 CONFIG_BOOTP_PREFER_SERVERIP=y
 CONFIG_POSITION_INDEPENDENT=y
 CONFIG_DISABLE_SDMMC1_EARLY=y
+CONFIG_CMD_PART=y
+CONFIG_PARTITION_UUIDS=y
+CONFIG_CMD_IMPORTENV=y
+CONFIG_BOOTDELAY=0
+CONFIG_CMD_EXPORTENV=y
-- 
2.17.1

